--- ./Makefile.PL.org	Fri Jan 25 01:42:27 2002
+++ ./Makefile.PL	Mon Jun  3 17:12:32 2002
@@ -2,10 +2,11 @@
 use strict;
 
 # tests demande ?
-my $ans = 
-  prompt("Do you want make smbclient tests ?\n".
-   "(you will be prompted for server / worgroup / share / user / password\n".
-   "to make read write access.", "no");
+my $ans = "no";
+#my $ans = 
+#  prompt("Do you want make smbclient tests ?\n".
+#   "(you will be prompted for server / worgroup / share / user / password\n".
+#   "to make read write access.", "no");
 if ($ans =~ /^y(es)?$/i) 
   {
     my $server = prompt("Server ?","localhost");
